src/form-engine.test.tsx

```diff
@@ -116,12 +109,14 @@ describe('Form component', () => {
     });
 
     await assertFormHasAllFields(screen, [
-      { fieldName: 'When was the HIV test conducted?', fieldType: 'date' },
+      { fieldName: 'When was the HIV test conducted? *', fieldType: 'date' },
       { fieldName: 'TB screening', fieldType: 'combobox' },
     ]);
 
     try {
       await findSelectInput(screen, 'Community service delivery point');
+      // await findSelectInput(screen, 'Community service');
+      // await screen.findByRole('combobox', { name: /community service delivery point/i });
```

Comment:
Was this intentional or don't we need them anymore?

Reply:
Fixed